clang/test/dpct/codepin/report/data_mismatch/CodePin_Report_ref.csv

```diff
@@ -0,0 +1,4 @@
+// CHECK: CodePin Summary
+// CHECK: Consistently APIs count, 1
+// CHECK: CUDA Meta Data ID, SYCL Meta Data ID, Type, Detail
+// CHECK: serial123-"var_2"-"Data"-"member_2"-"Data",serial123-"var_2"-"Data"-"member_2"-"Data",Data value, [VALUE MISMATCH] the CUDA value "bb123" differs from the SYCL value "ccc".
```

Comment:
Suggestion: Keep the CUDA MetaData ID a simple string that can be used to identify the file and put the detailed location info into the "detail" field.